.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,13 +4,16 @@ about: Create a report to help us improve
 
 ---
 
-扩展版本 (extension version): 
-浏览器版本 (browser version): 
-操作系统 (operating system): 
-问题描述 (problem description): 
- 
-实际行为 (actual behavior): 
- 
-期望行为 (expected behavior): 
- 
-重现步骤 (reproduce step):
+扩展版本/Extension Version:
+
+浏览器版本/Browser Type and Version (Chrome Version 70.0.3538.110 (Official Build) (64-bit); Firefox Version 63.0.3; etc.):
+
+操作系统/Operating System and Version (Windows 10 Pro Version	10.0.17134 Build 17134; Ubuntu 18.04 LTS; Debian Stretch V9; etc.):
```

Comment:
To avoid misunderstand please remove the unneeded information here. All of the issues provide enough information before. I don't think that this template needs to be modified.